test/00_transform/Norm.cu

```diff
@@ -85,8 +85,16 @@ TYPED_TEST(NormTestFloatTypes, VectorL1)
   this->pb->RunTVGenerator("vector_l1");
   this->pb->NumpyToTensorView(this->in_v, "in_v");
   this->pb->NumpyToTensorView(this->out_v, "out_v");
+
+  auto redOp = vector_norm(this->in_v, NormOrder::L1);
+
+  EXPECT_TRUE(redOp.Rank() == this->out_v.Rank());
+  for(int i = 0; i < redOp.Rank(); i++) {
+    EXPECT_TRUE(redOp.Size(i) == this->out_v.Size(i));
+  }
+
   // example-begin vector-norm-test-1
-  (this->out_v = vector_norm(this->in_v, NormOrder::L1)).run(this->exec);
+  (this->out_v = redOp).run(this->exec);
```

Comment:
This changes the example in the docs so it no longer shows the norm. This should probably be a separate test without a docs example

Reply:
fixed